src/wovnio/html/HtmlConverter.php

```diff
@@ -55,6 +55,18 @@ public function insertSnippetAndHreflangTags($html, $add_fallback_mark)
         return $converted_html;
     }
 
+    public function insertHtmlLangAttribute($html, $lang_code)
+    {
+        if (preg_match('/<html\s?.*?>/', $html, $matches)) {
+            $html_open_tag = $matches[0];
+            $replacement = $matches[0];
+            $replacement = preg_replace('/lang=["\']?[a-zA-Z-]*["\']?/', '', $replacement); // remove exsiting lang attribute
```

Comment:
if this is only getting called if we are not making an API call, should we be keeping the existing lang tag instead? We aren't actually changing the language in this case.

Reply:
Now, I think library should always insert default lang attr.
Because, if HTML translated properly, html-swapper changed lang attr.

Reply:
I'm more thinking if the request is for a default lang and not using html swapper, and they have `lang=fr` in their HTML already. Should we keep it or not?

Reply:
I guess that case is kind of client's mistake for setting WOVN.php.
But I agree to not force our lang setting.
I will change to not overwrite lang attr.